### PR TITLE
Minor changes

### PR DESCRIPTION
**Pull request by fnxpt**

### CODE REVIEW

File: step.yml [hunk 1]
```diff
@@ -42,8 +44,14 @@ inputs:
   opts:
     title: "Path to deliverable"
     description: |-
-      Path to deliverable, apk for android, xcarchive for iOS
+      Path to deliverable, apk/aar for android, xcarchive for iOS
```
Comment (drewhamilton) [inline step.yml:47]:
```suggestion
      Comma-delimited paths to deliverables: APK/AARs for android, xcarchive for iOS
```

File: step.yml [hunk 2]
```diff
@@ -42,8 +44,14 @@ inputs:
...
     is_required: true
+- signing_path: null
+  opts:
+    title: "Path to deliverable to sign"
```
Comment (drewhamilton) [inline step.yml:51]:
```suggestion
    title: "Paths to deliverables to sign"
```

File: step.yml [hunk 3]
```diff
@@ -42,8 +44,14 @@ inputs:
...
+    description: |-
+      Path to deliverable to sign, default value is deliverable_path
```
Comment (drewhamilton) [inline step.yml:53]:
```suggestion
      Comma-delimited paths to deliverables to sign, default value is deliverable_path
```

File: step.sh [hunk 4]
```diff
@@ -9,6 +9,9 @@ if [ -z $ARTIFACTORY_USERNAME ] || [ -z $ARTIFACTORY_PASSWORD ]; then
   exit 1
 fi
 
+if [ -z $signing_path]; then
+  options="$options --signArtifacts $signing_path"
+fi
```
Comment (drewhamilton) [inline step.sh:14]: In the description you say it defaults to `deliverable_path`, but here it looks like it's omitted if empty.

**Review by drewhamilton — COMMENTED** [review]
Added some suggestions on phrasing, and seeking clarification on the behavior if `signing_path` is empty.

File: step.sh (hunk 4 above)
Comment (fnxpt) [reply step.sh:14]: yes, its done internally on the other script

File: step.yml [hunk 5]
```diff
@@ -42,8 +44,14 @@ inputs:
   opts:
     title: "Path to deliverable"
```
Comment (drewhamilton) [inline step.yml:45]: Oops one more
```suggestion
    title: "Paths to deliverables"
```

**Review by drewhamilton — APPROVED** [review]

**Pull request merged**